style/README.md

```diff
@@ -346,3 +346,12 @@ Testing
   appear in the class.
 
 [Imperative mood]: http://en.wikipedia.org/wiki/Imperative_mood
+
+Legacy Code
```

Comment:
I think 99% of the time, we shouldn't be deprecating methods at all. We've already got too much of a bad habit of leaving `TODO: change this` comments in the code around for a long time, and I don't see marking methods as deprecated any differently. Let's just make the change if something needs to change.

The only exception would be API endpoint used outside the company (or perhaps used by the mobile app), but those won't be helped by the Rails `deprecate` method.